src/main/java/pl/kamann/services/email/EmailDetails.java

```diff
@@ -0,0 +1,15 @@
+package pl.kamann.services.email;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.Locale;
+
+@AllArgsConstructor
+@Getter
+public class EmailDetails {
+    private String to;
+    private String type;
+    private Locale userLocale;
+    private String content;
+}
```

Comment:
We can replace this class with a record since it is immutable and provides getters, equals/hashCode, and a constructor by default